### PR TITLE
Download proto

### PR DESCRIPTION
**Pull request by artursouza**
# Description

Downloads proto files into target dir and generate code from there. It keeps a fixed reference and can easily be updated.

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #13 

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [X] Code compiles correctly
* [ ] Created/updated tests
* [X] Extended the documentation

### CODE REVIEW

**Comment by artursouza** [comment]
Only merge after code freeze is over. This is not a needed fix to release 0.4.

File: pom.xml [hunk 1]
```diff
@@ -17,6 +17,8 @@
     <grpc.version>1.25.0</grpc.version>
     <protobuf.version>3.11.0</protobuf.version>
     <protoc.version>3.10.0</protoc.version>
+    <dapr.proto.url>https://raw.githubusercontent.com/dapr/dapr/e540a7166aeaf115773ccc4c7a1056ae7eed073b/pkg/proto/dapr/dapr.proto</dapr.proto.url>
```
Comment (youngbupark) [inline pom.xml]: JFYI - This url is snapshot, which means it will not be a up-to-date file

Comment (artursouza) [reply pom.xml]: This is by design. We want to have a fixed version. So this build has predictable behavior.

Comment (youngbupark) [reply pom.xml]: How about the version tag instead of hash? e.g. https://raw.githubusercontent.com/dapr/cli/v0.3.0/cmd/dapr.go

Comment (artursouza) [reply pom.xml]: Yes, that is preferred but this ref is for the 0.4 milestone and we did not have one at that time.

Comment (artursouza) [reply pom.xml]: Updated to use v0.4.0

**Pull request merged**